README.md

```diff
@@ -1,5 +1,7 @@
 # CSCH Megabot
 
+https://img.shields.io/github/license/cscareerhub/megabot
```

Comment:
based on what I see when I click the link I don't think this will work. Can we use this:

```suggestion
[![License: MIT](https://img.shields.io/badge/license-MIT-green.svg)](https://opensource.org/licenses/MIT)
```

Reply:
This dynamically gets license

```suggestion
[![License](https://img.shields.io/github/license/cscareerhub/megabot)](LICENSE.md)
```

Reply:
Yeah, that's better 👍